content/en/community/hackathons/sessions/debugging/content/Practical-Work_08.-Give-Us-Feedback.md

```diff
@@ -1,3 +1,3 @@
 We want to know how to make the next sessions better.
-For this we need your [feedback](https://forms.gle/9EuzgL1n244Mvqfq8).
+For this, we need your [feedback](https://docs.google.com/forms/d/e/1FAIpQLSdpyEpCW0yxJBSDa7Iq95JUCe5YNnRpxOD31ol4ezpKWnqrhA/viewform).
```

Comment:
This change should not happen here, since it is event-specific, and sessions in the `community/hackathons/sessions` will be added to every event.
The change should be made in the usoc'22 sessions index file.
See [this file](https://github.com/unikraft/docs/pull/96/files#diff-8f5b68fbca8819baba2f15887b55e7bb70cca9c17541b4c6fbe5ab66b39ff844) for reference.